src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -135,6 +144,14 @@ public void parse_optionalFieldsMissing_success() {
         Person expectedPerson = new PersonBuilder(AMY).withTags().build();
         assertParseSuccess(parser, NAME_DESC_AMY + PHONE_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY,
                 new AddCommand(expectedPerson));
+
+        // blank phone number
+        Person expectedPersonBlankPhone = new PersonBuilder(DOMINIC).withTags().build();
+        assertParseSuccess(parser, NAME_DESC_DOMINIC + PHONE_DESC_DOMINIC + EMAIL_DESC_DOMINIC
+                        + ADDRESS_DESC_DOMINIC, new AddCommand(expectedPersonBlankPhone));
+        // no phone number tag
```

Comment:
again minor nit but you could leave a line between 151 and 152 to be consistent with other parts?